probe/endpoint/conntrack.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 const (
-	modules          = "/proc/modules"
-	conntrackModule  = "nf_conntrack"
+	// From https://www.kernel.org/doc/Documentation/networking/nf_conntrack-sysctl.txt
+	// Check a tcp-related file for existence since we need tcp tracking
+	procFileToCheck  = "sys/net/netfilter/nf_conntrack_tcp_timeout_close"
```

Comment:
To be more precise, this confirms it's been part of conntrack since the first commit https://github.com/torvalds/linux/commit/9fb9cbb1082d6b31fb45aa1a14432449a0df6cf1#diff-fbdc7dc5c7c4089799324f79fd85d799R543